.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           function _make() { make -j$(nproc) -s O=out $*; }
           function download_plug_and_trust() { mkdir -p $HOME/se050 && git clone --single-branch -b v0.4.2 https://github.com/foundriesio/plug-and-trust $HOME/se050/plug-and-trust || (rm -rf $HOME/se050 ; echo Nervermind); }
 
-          function download_scp_firmware() { git clone --single-branch https://github.com/ARM-software/SCP-firmware.git $HOME/scp-firmware || echo Nervermind; }
+          function download_scp_firmware() { git clone -n --single-branch https://github.com/ARM-software/SCP-firmware.git $HOME/scp-firmware && git -q -C $HOME/scp-firmware checkout v2.13.0 || echo Nervermind; }
```

Comment:
for maintenance, maybe you would prefer:
```
          export SCP_FIRMWARE_REVISION=v2.13.0
          (...)
          function download_scp_firmware() { git clone ... && git -q -C $HOME/scp-firmware checkout $SCP_FIRMWARE_REVISION || echo Nervermind; }

Reply:
No, let's keep things more compact.